tests/validation/tests/v3_api/test_ingress.py

```diff
@@ -332,10 +333,15 @@ def test_ingress_rule_with_only_path():
 
     host = ""
     path = "/service2.html"
-    rule = {"host": host,
-            "paths": [{"workloadIds": [workload.id],
-                       "targetPort": TEST_IMAGE_PORT}]}
-    p_client.create_ingress(name=name,
+    if RANCHER_K3S_VERSION != "":
+        rule = {"host": host,
+                "paths": [{"workloadIds": [workload.id],
+                           "targetPort": TEST_IMAGE_PORT, "pathType": "Prefix", "path": "/"}]}
+    else:
+        rule = {"host": host,
+                "paths": [{"workloadIds": [workload.id],
+                           "targetPort": TEST_IMAGE_PORT}]}
+        p_client.create_ingress(name=name,
```

Comment:
an ingress will only be created when the K3s version is set here. Is that the intention?

Reply:
No. If the k3s version is set the ingress will be created with `"pathType": "Prefix", "path": "/"` else the ingress will be created without pathType and Prefix.

Reply:
So, shouldn't this ingress be outside of the else part and be run for both conditions?

Reply:
Right, I corrected the code.